src/test/blockencodings_tests.cpp

```diff
@@ -68,9 +68,8 @@ BOOST_AUTO_TEST_CASE(SimpleRoundTripTest)
 {
     CTxMemPool pool;
     TestMemPoolEntryHelper entry;
+    LOCK2(cs_main,pool.cs);
```

Comment:
I'm extremely hyper-sensitive to cs_main additions; do we know exactly what the lock is being used for in this context?

Reply:
I will recheck this change,maybe was a mistake when porting over the mempool test fixes

Reply:
might not be a porting mistake.  They like to sprinkle around cs_main like an american likes their salt.

Reply:
It is just in tests, I'm personally not too worried about it.